tests/vmi_configuration_test.go

```diff
@@ -1835,6 +1836,124 @@ var _ = Describe("Configurations", func() {
 		})
 	})
 
+	Context("Block size configuration set", func() {
```

Comment:
It's possible this test is having issues with running in parallel due to what `CreateBlockVolumePVandPVC` does. I'm looking into this. I think there's an issue with oru test suite here.